server/types/transferwise.ts

```diff
@@ -178,3 +178,44 @@ export type CurrencyPair = {
     fixedTargetPaymentAllowed: boolean;
   }[];
 };
+
+export type Balance = {
```

Comment:
Kudos for adding the types for Transferwise :+1:

Reply:
Thanks!

Reply:
I'm not sure if this something we should do because we'll eventually end up giving maintenance to external code.
Should we spawn a library to deal with TW so we can have community support?

Reply:
Would make sense to me, if you want to start one feel free to do so